frontend/src/pages/distributedWorkloads/global/GlobalDistributedWorkloads.tsx

```diff
@@ -33,7 +34,6 @@ const GlobalDistributedWorkloads: React.FC<GlobalDistributedWorkloadsProps> = ({
       />
     );
   }
-
   if (activeTab.projectSelectorMode !== null) {
```

Comment:
Come to think of it @gitdallas we could also remove all instances of the `projectSelectorMode` for these tabs since it'll always follow the `singleProjectOnly` logic now